Networking/Networking/Model/ShippingLabel/Packages/Carriers and Rates/ShippingLabelPackageSelected.swift

```diff
@@ -26,6 +34,7 @@ public struct ShippingLabelPackageSelected: Equatable, GeneratedFakeable {
     }
 
     public init(customPackage: ShippingLabelCustomPackage, totalWeight: Double, customsForm: ShippingLabelCustomsForm?) {
+        self.id = customPackage.title
```

Comment:
From what I see when inspecting API requests on the web app, it seems that the custom packages always have ID `default_box`. Should we do the same on the app?

Reply:
I'll refactor this part of the code in the next PR, because with the multi-package support is no more useful. If the user will choose the same custom package for two different packages, it means that the API will return just one key with the name of the custom package, and it's wrong (because we have two different packages, with different specs, but with the same package).